hw/ip/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson

```diff
@@ -0,0 +1,149 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
this file is almost identical to the original `flash_ctrl_sim_cfg.hjson`, except removing `tool`